deploy/cluster_role_binding_createns.yaml

```diff
@@ -0,0 +1,22 @@
+#
+#  Copyright (c) 2012-2019 Red Hat, Inc.
+#    This program and the accompanying materials are made
+#    available under the terms of the Eclipse Public License 2.0
+#    which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+#  SPDX-License-Identifier: EPL-2.0
+#
+#  Contributors:
+#    Red Hat, Inc. - initial API and implementation
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: che-operator-create-namespaces
+subjects:
+  - kind: ServiceAccount
+    name: che-operator
+    namespace: che
```

Comment:
I don't understand how this can fit the OLM and OperatorHub use-cases, since in these cases we don't control at all the namespace in which the operator will be installed and the `CheCluster` be created to trigger the Che installation.
But even `chectl`-based installation allows configuring the namespace where operator and Che server will be installed.
So I don't understand how a namespaced cluster role binding makes sense here.

Shouldn't this role binding be configured by the operator itself since knows in which namespace it is installed, and also controls where a given CheCluster will be installed, and with which workspace creation strategy it is configured.

Reply:
OLM generate own clusterroles and roles using package yaml and set up namespace defined in the group yaml: https://github.com/eclipse/che-operator/blob/master/olm/olm.sh#L127

Reply:
file removed

Reply:
Thanks @AndrienkoAleksandr !